patches/minecraft/net/minecraft/pathfinding/WalkNodeProcessor.java.patch

```diff
@@ -1,10 +1,42 @@
 --- a/net/minecraft/pathfinding/WalkNodeProcessor.java
 +++ b/net/minecraft/pathfinding/WalkNodeProcessor.java
-@@ -463,9 +463,11 @@
+@@ -392,16 +392,21 @@
+    }
  
+    public PathNodeType func_186330_a(IBlockReader p_186330_1_, int p_186330_2_, int p_186330_3_, int p_186330_4_) {
+-      return func_237231_a_(p_186330_1_, new BlockPos.Mutable(p_186330_2_, p_186330_3_, p_186330_4_));
++      return func_237231_a_(p_186330_1_, new BlockPos.Mutable(p_186330_2_, p_186330_3_, p_186330_4_), this.field_186326_b);
+    }
+ 
++   @Deprecated
+    public static PathNodeType func_237231_a_(IBlockReader p_237231_0_, BlockPos.Mutable p_237231_1_) {
++      return func_237231_a_(p_237231_0_, p_237231_1_, null);
++   }
++
++   public static PathNodeType func_237231_a_(IBlockReader p_237231_0_, BlockPos.Mutable p_237231_1_, @Nullable MobEntity mobEntity) {
```

Comment:
This and the other method you add an argument to need exec/srg lines so that it actually generates properly if people ever name the field/parameters.

Reply:
ok, I hope I did it right. 🙏